var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -112,6 +112,7 @@ class Opencv(CMakePackage, CudaPackage):
                  tag="{0}".format(cv),
                  when='@{0}+cuda'.format(cv))
 
+    depends_on('cmake@:3.11', when='@:3.2', type='build')
```

Comment:
Can you explain this?

Reply:
cmake 3.12 is changed python detection code.
I'm backport for cmake 3.12 or later.